backend/websocketConnect.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import FastAPI, WebSocket, WebSocketDisconnect
+import asyncio
+
+biostepFeedback = FastAPI()
```

Comment:
It will be helpful to name this instance as `app`. if we had many apps we could have named this as `biostepFeedbackApp`, it just helps for the word app to show up. Anyone who's familiar with FastAPI and wants to contribute to this code will be looking out for `app` in `main.py`.
Kindly adjust this.